addons/redis/templates/componentdefinition-redis-sentinel.yaml

```diff
@@ -76,6 +76,23 @@ spec:
           name: default
           optional: false
           password: Required
+    ## the default password of redis connection
+    - name: REDIS_DEFAULT_PASSWORD
```

Comment:
rename REDIS_DEFAULT_PASSWORD to  REDIS_SENTINEL_PASSWORD。this is the password of redis for redis sentinel connection, not  the default redis password